### PR TITLE
add description, license and collaborators sub-pages

### PR DESCRIPTION
**Pull request by SeverS**
## Description
- added description page
- added license page (license description is still wip)
- added collaborators page

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Review by didd — DISMISSED** [review]

File: extensions/apps/extensions/src/components/app-routes/index.tsx [hunk 1]
```diff
@@ -188,11 +203,34 @@ const devInfoRoute = createRoute({
...
       <CatchBoundary getResetKey={() => 'collaborators_reset'} errorComponent={RouteErrorComponent}>
-        <CollaboratorsPage appId={appId} />
+        <Suspense>
+          <Await promise={extensionById}>
+            {data => (
+              <CollaboratorsPage
+                extensionLogo={selectExtensionLogo(data)}
+                extensionName={selectExtensionName(data)}
+                extensionDisplayName={selectExtensionDisplayName(data)}
+                collaborators={selectExtensionCollaborators(data)}
```
Comment (quininez) [inline extensions/apps/extensions/src/components/app-routes/index.tsx:231]: on the app model only the dids for collaborators are saved, so you need to fetch the profile data before passing it here, I created a new hook in my PR called use-profiles-list that can be used for this

Comment (SeverS) [reply extensions/apps/extensions/src/components/app-routes/index.tsx:231]: check this out: https://feat-extension-page-sub-1.akasha-world-framework.pages.dev/@akashaorg/app-extensions/info/complexity-app

Comment (quininez) [reply extensions/apps/extensions/src/components/app-routes/index.tsx:231]: ok, I see that one of the profiles has also name, but I don't understand where it's coming from by looking at the code, it doesn't look like you are fetching profile data anywhere,  and on the model it is not stored. I will take a more in depth look after I fix my PR

Comment (SeverS) [reply extensions/apps/extensions/src/components/app-routes/index.tsx:231]: dark magix

**Review by quininez — APPROVED** [review]

**Review by didd — APPROVED** [review]

**Pull request merged**